config/minimalist-templates/popups_templates.yaml

```diff
@@ -0,0 +1,531 @@
+                    #     - position: absolute
+                    #     - left: 64%
+                    #     - top: >
+                    #         [[[
+                    #           var bri = states[entity.entity_id].attributes.brightness;
+                    #           return Math.round(bri ? (80-(72*bri/256)) : '80') + '%';
+                    #         ]]]
+                    #     - height: 25px
+                    #     - width: 45px
+                    #     - font-size: 11px
+                    #     - line-height: 25px
+                    #     - color: white
+                  custom_fields:
+                    item1:
+                      card:
+                        type: "custom:slider-card"
+                        entity: "[[[ return entity.entity_id ]]]"
+                        radius: 20px
+                        rotate: 270
+                        containerHeight: 320px
+                        width: 250px
+                        height: 100px
+                        top: 100px
+                        left: 17%
+                        thumbHorizontalPadding: 0px
+                        thumbWidth: 0px
+                        mainSliderColor: rgba(var(--color-theme),0.1)
+                        mainSliderColorOff: rgba(var(--color-theme),0.1)
+                        secondarySliderColor: var(--color-theme)
+                        secondarySliderColorOff: var(--color-theme)
+                        card_mod:
+                          style: |
+                            ha-card {
+                              background: none !important;
+                              box-shadow: none !important;
+                            }
+                            .off {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
+                            .on {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
+                    # item2: >
+                    #   [[[
+                    #     var bri = states[entity.entity_id].attributes.brightness;
+                    #     return Math.round(bri ? (bri/256*100) : '0') + '%';
+                    #   ]]]
+                #####
+                color:
+                  type: custom:light-entity-card
+                  entity: "[[[ return entity.entity_id ]]]"
+                  persist_features: true
+                  brightness: false
+                  color_temp: false
+                  white_value: false
+                  hide_header: true
+                  effects_list: false
+                  card_mod:
+                    style:
+                      ".control":
+                        $: |
+                          #interactionLayer{ z-index: 1; filter: opacity(0.6);}
+                          #backgroundLayer{
+                            filter: blur(20px);
```

Comment:
you can use the `smooth_color_wheel` property from the `light-entity-card` instead of blurring the wheel yourself.